app/core/error_handlers.py

```
@@ -0,0 +1,36 @@
+
+def format_error_response(status_code: int, detail: str, error_type: str = "error"):
+    return {"status": "error", "error_type": error_type, "detail": detail}
+
+async def http_exception_handler(request: Request, exc: HTTPException):
+    logger.error(f"HTTPException: {exc.detail} | Path: {request.url}")
+    return JSONResponse(
+        status_code=exc.status_code,
+        content=format_error_response(exc.status_code, exc.detail, "http_exception"),
+    )
+
+async def validation_exception_handler(request: Request, exc: RequestValidationError):
```

Ruff (0.8.2)

Missing return type annotation for public function `validation_exception_handler`

(ANN201)